javascript-modal/script.js

```diff
@@ -0,0 +1,34 @@
+var noPopup = true;
+var $bluebutton = document.querySelector('.blue-button');
+var $redbutton = document.querySelector('.red-button');
+var $modal = document.querySelector('.modal');
+var $background = document.querySelector('.background-default');
+
+$bluebutton.addEventListener('click', function () {
+  if (noPopup) {
```

Comment:
Doesn't look like you change the value of this `noPopup` variable anywhere, you can remove it and these conditionals, just the work within the conditional code block is necessary.